qiskit/transpiler/passmanager_config.py

```diff
@@ -95,6 +95,9 @@ def __init__(
         self.target = target
         self.hls_config = hls_config
         self.qubits_initially_zero = qubits_initially_zero
+        # Stores whether the basis gates are Clifford+T,
+        # in which case we use stage manager plugins adapted to Clifford+T.
+        self._is_clifford_t = False
```

Comment:
For now this works, but if we end up with different basis gate sets for FTQC (such as PBC) then we'll need to make this more extensible by maybe determining the basis gate mode and having separate pass managers for these

Reply:
Agreed!